service/header/store.go

```diff
@@ -76,6 +64,22 @@ func newStore(ds datastore.Batching) (*store, error) {
 	}, nil
 }
 
+func (s *store) Init(_ context.Context, initial *ExtendedHeader) error {
```

Comment:
It will be necessary eventually once we update to a newer Datastore(https://github.com/ipfs/go-datastore/blob/master/datastore.go#L59), so every write will require a context there.

Note that this is related to #436.